src/plugin/transform.ts

```diff
@@ -53,19 +55,20 @@ export function transform({
 
             const token = state.push(TOKEN_TYPE, TAG, 0);
             const htmlBlockId = generateHtmlBlockId();
+            const className = [BLOCK_NAME, containerClasses].filter(Boolean).join(' ');
+            const resultContent = withContentWrapper
+                ? `<div class="${BLOCK_NAME}__wrapper">${content}</div>`
+                : content;
```

Comment:
what is wrapper needed for?

Reply:
it is no longer relevant, I removed the wrapper, changed the resize logic, now I'm looking at the height of the body